app/src/main/java/com/onelist/dto/Category.kt

```diff
@@ -1,6 +1,9 @@
 package com.onelist.dto
 
-data class Category(val categoryID: Int, var name: String) {
+data class Category(
+    val categoryID: Int,
+    var name: String,
```

Comment:
Although not needed for the code to work, it is best practice to leave trailing commas for future development of a dto. I went ahead and update all of the dtos to include trailing commas

Link to source: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas